### ISSUE TITLE
Show nicer error if linter executable doesn't exist

### ISSUE THREAD

**Issue by Eschon** [AtomLinter/Linter#612]
I already know that the problem is that csslint is not at the specified path. But I thought it might be a good Idea to show some kind of warning instead of throwing an exception. 

**Atom Version**: 0.211.0
**System**: Mac OS X 10.10.3
**Thrown From**: [linter](https://github.com/AtomLinter/Linter) package, v1.0.4

### Stack Trace

Uncaught Error: ENOENT: no such file or directory, stat '/usr/local/bin/csslint'

```
At fs.js:846

Error: ENOENT: no such file or directory, stat '/usr/local/bin/csslint'
  at Error (native)
  at fs.statSync (fs.js:846:18)
  at Object.fs.statSync (ATOM_SHELL_ASAR.js:206:16)
  at LinterCsslint.Linter._cachedStatSync (/Users/egon/.atom/packages/linter/lib/linter.coffee:87:17)
  at LinterCsslint.Linter.getCmdAndArgs (/Users/egon/.atom/packages/linter/lib/linter.coffee:104:16)
  at LinterCsslint.Linter.lintFile (/Users/egon/.atom/packages/linter/lib/linter.coffee:145:24)
  at /Users/egon/.atom/packages/linter/lib/legacy.coffee:50:10
  at FSReqWrap.oncomplete (fs.js:77:15)

```

### Commands

```
     -8:36.1.0 core:delete (atom-text-editor.editor.is-focused)
     -8:26.2.0 core:copy (atom-text-editor.editor.is-focused)
     -8:24.9.0 core:paste (atom-text-editor.editor.is-focused)
  2x -8:17.1.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -1:07.9.0 editor:newline (atom-text-editor.editor.is-focused)
  9x -1:02.5.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -0:58.1.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
 10x -0:50.4.0 core:move-down (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:45.2.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:44 core:move-right (atom-text-editor.editor.is-focused)
     -0:39.2.0 core:copy (atom-text-editor.editor.is-focused)
     -0:36.7.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:36.5.0 core:paste (atom-text-editor.editor.is-focused)
     -0:32.4.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:11.7.0 core:save (atom-text-editor.editor.is-focused)
     -0:06.1.0 core:copy (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "solarized-dark-syntax"
    ],
    "disabledPackages": [
      "image-view",
      "background-tips",
      "language-go",
      "language-less",
      "language-property-list",
      "language-todo",
      "language-toml",
      "spell-check",
      "bookmarks",
      "welcome",
      "autoflow",
      "markdown-preview",
      "go-to-line",
      "language-c",
      "language-csharp",
      "language-java",
      "language-objective-c",
      "language-perl",
      "language-php",
      "language-python",
      "package-generator",
      "linter-tidy"
    ]
  },
  "linter": {
    "showInfoMessages": true,
    "subtleLinterErrors": [
      "jscs",
      "jshint"
    ],
    "clearOnChange": true,
    "showHighlighting": false
  }
}
```

### Installed Packages

```coffee
# User
atom-handlebars, v0.5.0
atom-ternjs, v0.6.4
color-picker, v2.0.7
linter, v1.0.4
linter-csslint, v0.0.13
linter-jscs, v1.12.0
linter-jshint, v0.1.6
linter-rubocop, v0.2.7
linter-xmllint, v0.0.6
pretty-json, v0.3.3
tabs-to-spaces, v0.11.0

# Dev
No dev packages
```

**Comment by steelbrain** [comment]
This is an error in the linter you're using and not in `linter` itself. You might want to do `npm install -g csslint`.
and if it still doesn't work, You might wanna open an issue there.

**Comment by Eschon** [comment]
@steelbrain As I said at the beginning of my post I already know that this error occurred because I didn't have csslint installed. But I thought it would be nicer to show a warning instead of throwing an exception.
Is this handled by linter or should I open an issue at linter-csslint?

**Comment by steelbrain** [comment]
Agreed.

**Comment by steelbrain** [comment]
Fix is up in #615
